### PR TITLE
GOV.UK Frontend v3

### PR DESCRIPTION
**Pull request by freesteph**
# Description
We're porting our services to the [new release of GOV.UK Frontend (v3)](https://github.com/alphagov/govuk-frontend/releases/tag/v3.0.0). Namely markup changes and some updates to the Gemfile to reduce our usage of the deprecated `sass` gem.

# Testing
1. build the branch;
2. run exploratory testing.

### CODE REVIEW

File: app/assets/stylesheets/utilities/colours.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
-$colour-black: govuk-colour("black");
-$colour-blue: govuk-colour("blue");
-$colour-light-blue: govuk-colour("light-blue");
-$colour-light-grey: govuk-colour("grey-1");
-$colour-turquoise: govuk-colour("turquoise");
-$colour-white: govuk-colour("white");
+$colour-black: govuk-colour('black');
+$colour-blue: govuk-colour('blue');
+$colour-light-blue: govuk-colour('light-blue');
+$colour-light-grey: govuk-colour('dark-grey');
```
Comment (phantas) [inline app/assets/stylesheets/utilities/colours.scss]: rename this to dark-grey, plz.

File: app/assets/stylesheets/utilities/colours.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+$colour-turquoise: govuk-colour('turquoise');
```
Comment (phantas) [inline app/assets/stylesheets/utilities/colours.scss]: is this used for a confirmation panel? If so, we should use green instead according to https://github.com/alphagov/govuk-frontend/releases/tag/v3.0.0

File: app/assets/stylesheets/utilities/colours.scss [hunk 3]
```diff
@@ -1,6 +1,6 @@
-$colour-black: govuk-colour("black");
```
Comment (phantas) [inline app/assets/stylesheets/utilities/colours.scss]: Can you please add a card to tech debt for refactoring and add an item to see if we can get rid of this file?

File: app/assets/stylesheets/utilities/colours.scss (hunk 2 above)
Comment (freesteph) [reply app/assets/stylesheets/utilities/colours.scss]: no I believe we use it for our flash messages, we don't sport any confirmation panel as far as I can tell.

File: app/assets/stylesheets/utilities/colours.scss (hunk 3 above)
Comment (freesteph) [reply app/assets/stylesheets/utilities/colours.scss]: Sure.

**Pull request merged**